hapic/hapic.py

```diff
@@ -445,7 +440,7 @@ def spec_yaml_view():
         )
 
         # Declare the swaggerui view
-        def api_doc_view():
+        def api_doc_view(*args, **kwargs):
```

Comment:
same question

Reply:
Same answer : Compatibility with any frameworks.

Reply:
this explanation should be added as a comment

Reply:
Done.